tests/README.md

```diff
@@ -91,14 +104,23 @@ For example look at the [block long haul test](/tests/block/k8s/longhaul/basicBl
  To run a longhaul test you can run any integration test with `-count` and `--load_parallel_runs` options
  e.g.
  ```
- go test -run TestK8sBlockLongHaul github.com/rook/rook/e2e/tests/block/k8s/longhaul --load_parallel_runs=20 -count=1000
+ go test -run TestK8sBlockLongHaul github.com/rook/rook/tests/longhaul --load_parallel_runs=20 -count=1000
  ```
+ The Longhaul test just like any other test is going to install and uninstall rook for each test run. If you want to run a
```

Comment:
Can we change the longhaul tests to not uninstall/install rook with every run? The purpose of longhaul is to see the same instance of rook staying up for long periods of time. I don't think we gain anything in longhaul by tearing down rook every time